src/app/app.component.ts

```diff
@@ -34,8 +34,8 @@ export class AppComponent {
       downloadUrl: './assets/images/04.png',
     },
     {
-      name: 'file_05.png',
-      type: 'image',
+      name: 'file_05.file',
```

Comment:
.file?

Reply:
Yeah, I changed one of the files to have an invalid type, so that it displays the 'not visible' placeholder instead